worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageProcessorImpl.java

```diff
@@ -70,6 +74,8 @@
     private final ThreadPoolTaskExecutor executor;
     private final int maxAttempts;
 
+    private ContractService contractService;
```

Comment:
Make final.

Not thrilled at introducing another service into the Coverage arena but we can revisit this another time.  This PR has dragged out way too long.